### PR TITLE
Changes to Tempest in RDO Ocata

### PR DESCRIPTION
**Pull request by ltoscano-rh**
Add a link to this article in the previous articles which described Tempest.

### CODE REVIEW

File: source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+This means less burden work required (no need to keep the fork) but also
+simplifies a bit the steps required to initialize and run Tempest tests,
+close to the process documented by Tempest upstream.
+
+## New configuration steps
+
+Configure the RDO repositories, then install the required packages:
+
+    $ yum -y install python-tempestconf openstack-tempest
```
Comment (chkumar246) [inline source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md]: $ it should be $sudo yum -y install openstack-tempest, python-tempestconf is a dependency of openstack-tempest so no need to install python-tempestconf.

Comment (ltoscano-rh) [inline source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md]: Oh, right. Waiting for other reviews before changing it.

File: source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    $ cd testingdir
+    $ discover-tempest-config --debug identity.uri $OS_AUTH_URL \
+          identity.admin_password  $OS_PASSWORD --create
+
+`discover-tempest-config` is the new name of the old `config_tempest.py`
+script and it accepts the same parameters.
+
+And that's it! Now it is possible to lists and run the tests a usual:
+
+    $ tempest run
```
Comment (chkumar246) [inline source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md]: What about giving a example something like $  tempest run --regex heat_integrationtests or  tempest run --regex "^neutron_vpnaas.tests.tempest.api\."

Comment (ltoscano-rh) [inline source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md]: The idea is "look at the upstream documentation, nothing changed here".

Comment (chkumar246) [inline source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md]: Looks, Ok then.

**Pull request merged**